docs/modules/reference/pages/configuration/ifttt-integration.adoc

```diff
@@ -0,0 +1,140 @@
+
+= IFTTT Integration
+
+The IFTTT service lets you combine web applications using simple conditional instructions.
+Each supported service has several triggers that you can use to trigger the actions of other services.
+This lets you, for example, change brightness and color of a smart bulb, and send messages or date to IoT devices.
+
+The {page-component-title} integration uses the Webhooks service, that lets you trigger actions when a specific web request is received.
+The basic operation is as follows: {page-component-title} polls for alarms and matches the alarm reduction key against a given filter and the alarm's associated nodes against a given category filter.
+For the resulting alarm, set the maximum severity and the total count is computed.
+If one of these values has changed compared to the last poll, one or more events specified for the computed maximum severity will be sent to IFTTT.
+
+== IFTTT configuration
+
+To use the IFTTT integration in {page-component-title} you need an IFTTT account.
+With this account you can create applets that combine a trigger with an action.
+In our case, we use the Webhooks service as the trigger and define the event name `OpenNMS`.
+After this step you can combine this trigger with any of the possible supported services and their actions.
+
+[[ga-ifttt-trigger-definition]]
+.Webhooks service trigger definition
+image::alarms/trigger-definition-small.png[]
+
+In your account service settings for the Webhooks service you find your key in the given service URL.
+In the following example this key is `X71dfUZsH4Wkl6cjsLjdV`.
+
+[[ga-ifttt-webhooks-settings]]
+.Webhooks service settings
+image::alarms/webhooks-settings-small.png[]
+
+On the side of {page-component-title}, you need a configuration that defines which event names to send on an alarm count or severity change.
+The configuration file `ifttt-config.xml` contains trigger packages.
+
+The operation is as follows:
+. {page-component-title} retrieves all alarms that have a node associated.
```

Comment:
This section is not showing up as steps in the final.
It's also not clear if this is steps the user can do to configure this behaviour of if this text is just describing something that happens.